src/api/.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config --auto-gen-only-exclude`
-# on 2020-03-26 14:38:56 +0000 using RuboCop version 0.80.1.
+# on 2020-03-27 13:48:30 +0000 using RuboCop version 0.80.1.
```

Comment:
After the upgrade rubocop started to complain about:
```
app/controllers/configurations_controller.rb:33:13: C: Rails/IndexWith: Prefer index_with over Hash[map { ... }].
    archs = Hash[xml['schedulers']['arch'].map { |a| [a, 1] }] if xml.dig('schedulers', 'arch').class == Array
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/controllers/configurations_controller.rb:34:13: C: Rails/IndexWith: Prefer index_with over Hash[map { ... }].
    archs = Hash[params['arch'].map { |a| [a, 1] }] if params['arch'].class == Array
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Not sure why it starts to complain now about it, since there are no changes in this file. But thats why I regenerated the rubocop_todo

Reply:
¯\_(ツ)_/¯